snowchange.py

```diff
@@ -123,19 +127,41 @@ def get_all_scripts_recursively(root_directory, verbose):
 
 def execute_snowflake_query(snowflake_database, query, verbose):
   con = snowflake.connector.connect(
-    user = os.environ["SNOWFLAKE_USER"],
-    account = os.environ["SNOWFLAKE_ACCOUNT"],
-    role = os.environ["SNOWFLAKE_ROLE"],
-    warehouse = os.environ["SNOWFLAKE_WAREHOUSE"],
-    database = snowflake_database,
-    region = os.environ["SNOWFLAKE_REGION"],
-    authenticator = os.environ["SNOWFLAKE_AUTHENTICATOR"],
-    password = os.environ["SNOWSQL_PWD"]
+    user=os.environ["SNOWFLAKE_USER"],
+    account=os.environ["SNOWFLAKE_ACCOUNT"],
+    role=os.environ["SNOWFLAKE_ROLE"],
+    warehouse=os.environ["SNOWFLAKE_WAREHOUSE"],
+    database=snowflake_database,
+    region=os.environ.get("SNOWFLAKE_WAREHOUSE") if os.environ.get("SNOWFLAKE_REGION") else '',  # no region if not specified
```

Comment:
Add comment referencing that "empty string" is the default in `snowflake.connector`